### DISCUSSION TITLE
Questions to get me started, using xDrip with Dexcom G6

### DISCUSSION THREAD

**Discussion by ClaudeVernier** [NightscoutFoundation/xDrip#2589] [General]
Hello
I'm sorry for first created an issue with this, I now discovered the discussions and wikis !!

I'm still a bit confused on where to start as I see several concepts I don't master (Dexcom ONE, Share Server...) and not knowing if they are still valid or available in France.

I use xDrip+ since months with FSL2, OOP, Nightscout and AAPS.
I just started using Dexcom G6 to improve BG readings in AAPS.

At the hospital, they had me install the Dexcom and Clarity apps to share readings with the "MyDiabby" app where they can follow my data. 

I read that since june, version 2022.06.02, xDrip can get data directly from the Dexcom.
When I look at my version (2122090208), it doesn't match to dates so I'm not sure of the version I have.
Attached is the version I'm referring to in xDrip, can I upgrade from the app or is it better to delete and reinstall ?

Could you help me getting started using G6 with xDrip without losing Clarity and official Dexcom app ?
Maybe there is a more recent documentation I didn't find.

Thanks for all the great work.

![Screenshot_20221223-223701_xDrip+ (1)](https://user-images.githubusercontent.com/3709997/210061422-db027d97-0b75-46df-ad06-dd8fedf3946c.jpg)

**Comment by Navid200** [comment]
Hi,

Would you please provide a link to where you downloaded that xDrip from?

This is where you can get the official xDrip from:  https://navid200.github.io/xDrip/docs/Download-xDrip

If the serial number of your Dexcom One transmitter starts with a 5, xDrip can collect from it directly.
If it starts with C, xDrip cannot collect from those yet.  Hopefully, soon.
https://navid200.github.io/xDrip/docs/Dexcom-One

If the serial number starts with C, at this point, your only option is to use the official Dexcom app and use the Companion app mode (https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html) in xDrip installed on the same phone.

**Reply by ClaudeVernier** [reply]
Hi,  
My phone had broken last summer and I reinstalled all quickly without taking notes but I always go to https://github.com/NightscoutFoundation/xDrip for packages.

The package on my phone as the name: xDrip-plus-20220902-f323b2b.apk which maps to the package here: https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.09.02

I believe I did try to update it at some point to use the Check Updates menu, could this have installed something bad ?

Does the name Dexcom ONE defines the phone-like device provided by Dexcom or is it wider than this ?
I only use my Samsung S8 phone with the Dexcom G6 transmitter, it confuses me, I found a Dexcom ONE web site in Belgium and switching to France, there was no more mentioning of Dexcom ONE.

Anyhow and even more troubling, my SN on the box is 8DJ708 !!  Ok, I read the documentation on: [Dexcom Basics](https://navid200.github.io/xDrip/docs/Dexcom-One), thanks.
I guess I'll want to have xDrip as a follower as I was planning to keep the original app.  

Maybe the SN is to be read on the xDrip UI, I updated the xDrip config to use Dexcom, I started a new sensor, entered the code and the start date which was December 23rd.  Now, xDrip waits for first readings but I will read further how to set it as follower.

Thanks a lot for your help, 
Claude


![20221230_151046](https://user-images.githubusercontent.com/3709997/210082096-9f2033d8-0dba-426b-b629-493cce00608c.jpg)

**Comment by Navid200** [comment]
The screenshot you have posted is of a G6 transmitter, not Dexcom One.  I misunderstood your reference to Dexcom One in your first post.

This is what the classic status page looks like:  https://navid200.github.io/xDrip/docs/xDrip-Version
Instead of posting that, you posted a screenshot of the update screen in French, is it?  So, again, I was confused by that.

If you want to use xDrip as a follower, these are the options:  https://navid200.github.io/xDrip/docs/How-to-follow.html

**Reply by ClaudeVernier** [reply]
Sorry for the confusion, it is not yet clear in my head.
About versions, yes, I thought the version number was on the update screen, here is my status page.

I don't understand the difference between Dexcom One and G6, the thing I do know is that I have G6, maybe that ONE is not available here.
I wanted to use Dexcom to have better readings in AAPS but can that be the case if xDrip is a follower ?

I could rephrase my need by saying: I want better readings in AAPS and also send my readings to the MyDiabby app that my doctor asks for.
Do you think that could work with xDrip as a follower ?

It seems the only choice would be using CamAPS as I don't want to rely on Internet for AAPS to get the readings, make sense ?
Currently, MyDiabby uses Clarity to get readings, this doesn't need to be synced online.

Otherwise, if I could get readings for AAPS and Clarity/MyDiabby without the Dexcom app and using xDrip as receiver, it could answer the need too.

For xDrip to be a receiver, does it need to be Dexcom ONE ?  Would it work with my Dexcom G6 transmitter ?

Thanks.
![Screenshot_20221230-202311_xDrip+](https://user-images.githubusercontent.com/3709997/210109913-c107567c-0590-4ca7-875e-75723c7d4338.jpg)

**Reply by Navid200** [reply]
xDrip can read from G6.  xDrip can read from Dexcom One.  
You have G6, not Dexcom One. 
Considering those, I hope we can stop talking about Dexcom One.

xDrip can send data to AAPS.
xDrip cannot upload to Clarity.

I don't know why you bring up CamAPS.

**Reply by ClaudeVernier** [reply]
I bring up CamAPS because I understood xDrip has to be follower if I want to keep Dexcom app.  
I read that only way for xDrip to be follower without using Internet was to use companion app and understood that CamAPS is companion app but again, I'm sure of nothing, just trying to figure out what I should do.

Thanks again for your patience.

**Reply by Navid200** [reply]
CamAPS is a closed loop app.  It controls a pump.  But, more importantly, it occupies one of the G6 transmitter Bluetooth slots.  It would make no sense to use it unless you absolutely need it because you have a pump that only works with CapAPS.

If Clarity is a requirement for you, this is your solution:  https://navid200.github.io/xDrip/docs/DexcomAppxDrip.html
If Clarity is only a preference but not a requirement, you can use xDrip and only xDrip.  And this explains how:  https://navid200.github.io/xDrip/docs/Starting-G6

**Reply by ClaudeVernier** [reply]
Ok, I'll try the BYODA until I meet the doctor again and if it turns out it is not a requirement, I'll switch to xDrip only.  Thanks a lot !
